helm-charts/docsum/values.yaml

```diff
@@ -64,8 +64,12 @@ llm-uservice:
   MAX_TOTAL_TOKENS: "2048"
   LLM_MODEL_ID: Intel/neural-chat-7b-v3-3
 
-# To override values in subchart tgi
+# To override values in tgi/vllm subcharts
 tgi:
+  enabled: true
+  LLM_MODEL_ID: Intel/neural-chat-7b-v3-3
```

Comment:
`MAX_INPUT_LENGTH` and `MAX_TOTAL_TOKENS` settings for tgi is missing during the rebase. We need to set them otherwise, CI test will fail.

Reply:
Thanks!   I added fix commit for that, and rebased to `main`.

Reply:
Thanks!   Resolving as I pushed fix few days ago.